src/haddock/modules/sampling/gdock/__init__.py

```diff
@@ -114,19 +111,32 @@ def _run(self):
         input_toml += f'B = \"{input["B"]}\"' + os.linesep
         input_toml += os.linesep
 
-        with working_directory(self.path):
-            with open('input.toml', 'w') as inp_fh:
-                inp_fh.write(input_toml)
+        # ===============
+        # Placeholder, this is not yet implemented in gdock
+        # input_toml += '[parameters]' + os.linesep
+        # input_toml += 'population_size = 100' + os.linesep
+        # input_toml += 'max_number_of_generations = 50' + os.linesep
+        # ===============
+
+        with open('input.toml', 'w') as inp_fh:
+            inp_fh.write(input_toml)
 
-            cmd = f'{sys.executable} {gdock_exec} --dry input.toml'
+        cmd = f'{sys.executable} {gdock_exec} --dry input.toml'
 
-            subprocess.call(cmd, shell=True)
+        subprocess.call(cmd, shell=True)
 
         # retrieve the structures
         output_structures = []
-        structure_folder = Path(self.path, 'gdock-integration/structures')
+        structure_folder = Path('gdock-integration/structures')
```

Comment:
Currently, `structure_folder` will be searched inside the `0*_gdock` folder in the `run_dir`. Make sure this is what you want, or if you need some `../data` first.

Reply:
yep took me a while to figure this out, as its now the next module can find the structures fine.

Reply:
After #219 each module runs on its own folder. We need to keep this in mind now when defining paths.